specs/withdrawals.md

```diff
@@ -55,18 +55,27 @@ We first describe the end to end flow of initiating and finalizing a withdrawal:
 ### On L2
 
 An L2 account sends a withdrawal message (and possibly also ETH) to the `L2ToL1MessagePasser` predeploy contract.
-   This is a very simple contract that stores the a hash of the withdrawal data.
+This is a very simple contract that stores the a hash of the withdrawal data.
 
 ### On L1
 
 1. A [relayer][g-relayer] submits the required inputs to the `OptimismPortal` contract. The relayer need
    not be the same entity which initiated the withdrawal on L2.
-   These inputs include the withdrawal transaction data, inclusion proofs, and a timestamp. The timestamp
+   These inputs include the withdrawal transaction data, inclusion proofs, and a block number. The block number
    must be one for which an L2 output root exists, which commits to the withdrawal as registered on L2.
-2. The `OptimismPortal` contract retrieves the output root for the given timestamp from the `OutputOracle`'s
-   `l2Outputs()` function, and performs the remainder of the verification process internally.
-3. If verification fails, the call reverts. Otherwise the call is forwarded, and the hash is recorded to prevent it from
-   from being replayed.
+1. The `OptimismPortal` contract retrieves the output root for the given block number from the `L2OutputOracle`'s
+   `getL2OutputAfter()` function, and performs the remainder of the verification process internally.
+1. If proof verification fails, the call reverts. Otherwise the call is forwarded, and the hash is recorded to
+   prevent it from being re-proven. Note that the withdrawal can be proven more than once if the corresponding
+   output root changes.
+1. After the withdrawal is proven, it enters a 7 day challenge period, allowing time for other network participants
+   to challenge its integrity.
```

Comment:
@clabby Is it really possible to challenge a withdrawal? I think only output root can be challenged.

Reply:
@norswap This is what I meant, but I see how the wording is a bit unclear- Should we change this up?

Maybe:
```suggestion
1. After the withdrawal is proven, it enters a 7 day challenge period, allowing time for other network participants
   to challenge the integrity of the corresponding output root.
```

Reply:
Sounds great! Sorry for the w-e ping btw, just sorting through old bookmarks 😅

Reply:
No problem! Appreciate the nit, good spot 😄